app/assets/javascripts/admin/addon/models/site-setting.js

```diff
@@ -39,6 +40,18 @@ export default class SiteSetting extends EmberObject.extend(Setting) {
     return ajax(`/admin/site_settings/${key}`, { type: "PUT", data });
   }
 
+  @alias("settingObjectHelper.overridden") overridden;
+  @alias("settingObjectHelper.computedValueProperty") computedValueProperty;
+  @alias("settingObjectHelper.computedNameProperty") computedNameProperty;
+  @alias("settingObjectHelper.validValues") validValues;
+  @alias("settingObjectHelper.allowsNone") allowsNone;
+  @alias("settingObjectHelper.anyValue") anyValue;
+
+  constructor() {
+    super(...arguments);
+    this.settingObjectHelper = new SettingObjectHelper(this);
+  }
+
```

Comment:
Any reason we can't do this in a field?

```suggestion
  settingObjectHelper = new SettingObjectHelper(this);
```